src/services/facilityServices.js

```diff
@@ -226,16 +249,17 @@ const assignFacilityComponents = async (facilityId, componentIds) => {
     return resStatus;
 }
 
-exports.getFacilityByRegionId = getFacilityByRegionId;
+exports.getFacilitiesByRegionId = getFacilitiesByRegionId;
 exports.getFacilityCountByStateId = getFacilityCountByStateId;
 exports.addFacility = addFacility;
 exports.updateFacility = updateFacility;
 exports.deleteFacilityById = deleteFacilityById;
 exports.destroyFacilityById = destroyFacilityById;
 exports.assignFacilityComponents = assignFacilityComponents;
+exports.getFacilitiesByStateId = getFacilitiesByStateId;
 
 // FOR DEBUGGING
-// getFacilityByRegionId(2).then(data => console.dir(data));
+// getFacilitiesByRegionId(2).then(data => console.dir(data));
 // deleteFacilityById(3).then(data => console.dir(data));
 // destroyFacilityById(4).then(data => console.dir(data));
 // assignFacilityComponents(1, [3]);

```

Comment:
Can we remove all this debug code?

Reply:
Wait what if a random developer want to debug and dont know how tho?\

Reply:
Yeah but this is just a sign of a lack of proper testing. We should probably invest in unit tests instead of something like this

Reply:
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA SAY THAT AERLIER AAAAAA

Alright lemme remove them and add another commit